x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/fleet/setup_page/index.tsx

```diff
@@ -38,84 +67,154 @@ export const SetupPage: React.FunctionComponent<{
             <FormattedMessage
-              id="xpack.ingestManager.setupPage.enableTitle"
-              defaultMessage="Enable Fleet"
+              id="xpack.ingestManager.setupPage.elasticsearchApiKeyFlagText"
+              defaultMessage="API key service. Set {apiKeyFlag} to {true} ."
+              values={{
+                apiKeyFlag: <EuiCode>xpack.security.authc.api_key.enabled</EuiCode>,
+                true: <EuiCode>true</EuiCode>,
+              }}
             />
-          </h2>
-        </EuiTitle>
-        <EuiSpacer size="xl" />
-        <EuiText color="subdued">
+          </RequirementItem>
+          <EuiSpacer size="m" />
+          <EuiCodeBlock isCopyable={true}>
+            {`xpack.security.enabled: true
+xpack.security.authc.api_key.enabled: true`}
+          </EuiCodeBlock>
+          <EuiSpacer size="l" />
           <FormattedMessage
-            id="xpack.ingestManager.setupPage.enableText"
-            defaultMessage="In order to use Fleet, you must create an Elastic user. This user can create API keys
-        and write to logs-* and metrics-*."
+            id="xpack.ingestManager.setupPage.missingRequirementsKibanaTitle"
+            defaultMessage="In your Kibana configuration, enable:"
+          />
+          <EuiSpacer size="l" />
+          <RequirementItem isMissing={missingRequirements.includes('tls_required')}>
+            <FormattedMessage
+              id="xpack.ingestManager.setupPage.tlsFlagText"
+              defaultMessage="Kibana security. Set. Set {securityFlag} to {true}. For development purposes, you can disable TLS by setting {tlsFlag} to {true} as an unsafe alternative."
+              values={{
+                securityFlag: <EuiCode>xpack.security.enabled</EuiCode>,
+                tlsFlag: <EuiCode>xpack.ingestManager.fleet.tlsCheckDisabled</EuiCode>,
+                true: <EuiCode>true</EuiCode>,
+              }}
+            />
+          </RequirementItem>
+          <EuiSpacer size="m" />
+          <RequirementItem
+            isMissing={missingRequirements.includes(
+              'encrypted_saved_object_encryption_key_required'
+            )}
+          >
+            <FormattedMessage
+              id="xpack.ingestManager.setupPage.encryptionKeyFlagText"
+              defaultMessage="Encryption key. Set {keyFlag} to use a secret key."
+              values={{
+                keyFlag: <EuiCode>xpack.reporting.encryptionKey</EuiCode>,
+              }}
+            />
+          </RequirementItem>
+          <EuiSpacer size="l" />
+          <FormattedMessage
+            id="xpack.ingestManager.setupPage.gettingStartedText"
+            defaultMessage="For more information, read our {link}."
+            values={{
+              link: (
+                <EuiLink
+                  href="https://www.elastic.co/guide/en/ingest-management/current/index.html"
+                  target="_blank"
```

Comment:
cc @hbharding what do you think of using `external={true}` for this EuiLink instead of `target="_blank"`? this will add an icon next to it too

![image](https://user-images.githubusercontent.com/1965714/87454933-889f8a00-c5b9-11ea-90a3-238744ba5ed9.png)

Reply:
Yeah good call thanks Jen. @nchaulet can you add that prop?